lib/StatusIndicator/styles.scss

```diff
@@ -0,0 +1,39 @@
+/* ==========================================================================
+   Config
+   ========================================================================== */
+
+$size-status-indicator: 1em !default;
```

Comment:
Such a minor thing but in other component styles we switch the name of the component with what it does 😂  e.g. `status-indicator-size` etc...